<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Page Object test" duration-ms="10093" started-at="2019-09-03T23:02:06Z" finished-at="2019-09-03T23:02:16Z">
    <groups>
    </groups>
    <test name="sample test" duration-ms="10093" started-at="2019-09-03T23:02:06Z" finished-at="2019-09-03T23:02:16Z">
      <class name="com.pack.common.tests.MyAccountTest">
        <test-method status="PASS" signature="initializeTestBaseSetup(java.lang.String, java.lang.String)[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69]" name="initializeTestBaseSetup" is-config="true" duration-ms="6628" started-at="2019-09-03T23:02:06Z" finished-at="2019-09-03T23:02:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://tutorialsninja.com/demo/index.php?route=common/home]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestBaseSetup -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69]" name="setUp" is-config="true" duration-ms="0" started-at="2019-09-03T23:02:13Z" finished-at="2019-09-03T23:02:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyLoginFunction()[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69]" name="verifyLoginFunction" duration-ms="1691" started-at="2019-09-03T23:02:13Z" finished-at="2019-09-03T23:02:15Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Sign In page title doesn't match expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Sign In page title doesn't match expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.pack.common.tests.MyAccountTest.verifyLoginFunction(MyAccountTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginFunction -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult)[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69]" name="takeScreenShotOnFailure" is-config="true" duration-ms="491" started-at="2019-09-03T23:02:15Z" finished-at="2019-09-03T23:02:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginFunction status=FAILURE method=MyAccountTest.verifyLoginFunction()[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.pack.common.tests.MyAccountTest@5c5a1b69]" name="tearDown" is-config="true" duration-ms="1250" started-at="2019-09-03T23:02:15Z" finished-at="2019-09-03T23:02:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.pack.common.tests.MyAccountTest -->
    </test> <!-- sample test -->
  </suite> <!-- Page Object test -->
</testng-results>
